src/components/LinksCard.tsx

```diff
@@ -24,7 +24,11 @@ export default function LinksCard({ links }: Props) {
   };
 
   return (
-    <Card pad="small" margin={{ vertical: "small" }}>
+    <Card
+      pad="small"
+      margin={{ vertical: "small" }}
+      style={{ boxShadow: "none" }}
```

Comment:
I removed the box shadow from here, as having nested cards doesn't look ideal. I think this should be converted to just a Box or custom component